src/main/java/org/isf/visits/gui/VisitView.java

```diff
@@ -317,11 +375,36 @@ private JButton getAddVisitSecondButton() {
 		}
 		return addSecondVisitButton;
 	}
+	
+	private JButton getDeleteSecondVisitButton() {
+		if (deleteSecondVisitButton == null) {
+			deleteSecondVisitButton = new JButton(MessageBundle.getMessage("angal.visit.removevisit.btn"));
+			deleteSecondVisitButton.setMaximumSize(new Dimension(VISIT_BUTTON_WIDTH, ALL_BUTTON_HEIGHT));
+			deleteSecondVisitButton.setHorizontalAlignment(SwingConstants.LEFT);
+			deleteSecondVisitButton.addActionListener(actionEvent -> {
+
+				int row = jTableSecond.getSelectedRow();
+				if (row < 0) {
+					MessageDialog.info(VisitView.this, MessageBundle.getMessage("angal.common.pleaseselectarow.msg"));
```

Comment:
The `MessageDialog.info()` method includes the `MessageBundle.getMessage()` so only pass `angal.common.pleaseselectarow.msg` and not the result of the lookup.

Reply:
Done, thanks!